corehq/apps/products/migrations/0001_initial.py

```diff
@@ -25,7 +25,7 @@ class Migration(migrations.Migration):
                 ('program_id', models.CharField(default=b'', max_length=100, null=True)),
                 ('cost', models.DecimalField(null=True, max_digits=20, decimal_places=5)),
                 ('units', models.CharField(default=b'', max_length=100, null=True)),
-                ('product_data', json_field.fields.JSONField(default={}, help_text='Enter a valid JSON object')),
+                ('product_data', jsonfield.fields.JSONField(default={})),
```

Comment:
This removes the old default ```help_text``` in ```json_field```.  Switching to ```jsonfield``` would require adding a new migration with an ```AlterField``` to get rid of the ```help_text```.  I choose just to remove it here; I'm not aware of any negative side-effects.